plone/schemaeditor/fields.py

```diff
@@ -117,17 +117,30 @@ def __init__(self, field):
 
     def __getattr__(self, name):
         if name == 'values':
-            return [term.value for term in (self.field.vocabulary or [])]
+            values = []
+            for term in (self.field.vocabulary or []):
+                if term.value != term.title:
+                    values.append('%s|%s' % (term.value, term.title))
```

Comment:
please use the new style string``.format()`` as for some reason we enforce it. code-analysis at least does not like the old style.

``'{0:s}|{1:s}'.format(term.value, term.title)``